less/index.less

```diff
@@ -17,43 +17,305 @@ article, aside, canvas, details, embed,
 * {
-    box-sizing: border-box;
+box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+height: 100%;
+width: 80%;
+margin: 0 auto;
+}
+
+// Variables
+
+@header-font: 18px;
+
+// Mixins
+
+.h4Stylz() {
+font-size: 18px;
+
+}
+
+.tdStylz() {
+padding-top: 10px;
+padding-left: 10px;
+}
+
+// Styling for the body
+
+body {
+width: 800px;
+
+@media(max-width: 500px) {
+
+		width: 500px;
+}
+}
+
+
+
+
+// Styling for the Nav element
+
+.navMenu {
+font-size: 12pt;
+
+
+
+nav {
+	display: inline-flex;
+	flex-direction: row;
+	justify-content: space-evenly;
+	margin-left: 55%;
+	width: 50%;
+	padding: 16px;
+}
+
+
+a {
+	text-decoration: none;
+	color: black;
 }
 
+@media (max-width: 500px) {
+	nav {
+		display: flex;
+		flex-direction: column;
+		margin: auto;
+
+		a {
+			font-size: @header-font;
+			height: 70px;
+			width: 500px;
+			text-align: center;
+			padding-right: 350px
+		}
+		
+	}
+}
+}
+
+
+
+// Styling for the header element
+
+header {
+	width: 100%;
+	height: 35em;
+	background: lightgrey;
+	border-radius: 5px;
+
+	@media (max-width: 500px) {
+		width: 80%;
+		height: 38em;
+		background: lightgrey;
+		border-radius: 5px;
+		margin-right: 25%;
+	
+	}
+
+	img {
+		width: 25em;
+		padding: 35px;
+
+		@media (max-width: 500px) {
+			margin: 10%;
+			margin-bottom: 0;
+			width: 70%;
+			padding: 7%;
+			padding-left: 14%;
+			
+		}
+	}	
+
+	h1 {
+		font-size: 30px;
+		margin-left: 30px;
+
+		@media (max-width: 500px) {
+			width: 50%;
+			margin: auto;
+			margin-left: 25%;
+			margin-bottom: 5%;
+			
+		}
+	}
+
+	h4 {
+		font-size: @header-font;
+		padding-bottom: 9px;
+		border-bottom: 1pt solid black;
+		margin: 2%;
+		margin-left: 6%;
+
+		@media (max-width: 500px) {
+			text-align: center;
+		}
+	}
+
+
+	p {
+		margin-left: 8%;
+
+		@media (max-width:500px) {
+			margin-left: 20%;
+			width: 60%;
+			text-align: center;
+			margin-top: 6%;
+			
+		}
+	}
+
+}
+
+// Styling for the SectionClass "bodyStuff" container
+
+.bodyStuff {
+	display: flex;
+	flex-direction: row;
+	width: 70%;
+	padding-top: 50px;
+
+	@media (max-width: 500px) {
+		display: flex;
+		flex-direction: column;
+	}
+
+	.resumeBody {
+		width: 70%;
+
+		@media (max-width: 500px) {
+			margin-bottom: 10%;
+    		margin-left: 25%;
+		}
+
+		h4 {
+			font-size: @header-font;
+
+			@media (max-width: 500px) {
+				text-align: center;
+				margin-bottom: 5%;
+			}
+
+			p {
+				width: 50%;
+				height: auto;
+			}
+		}
+
+	}	
+	
+}
+
+// Styling for the .goodStuff table
+
+.goodStuff {
+width: 320px;
+margin: 0px;
+padding-left: 5px;
+
+table {
+	width: 400px;
+	height: 200px;
+	margin-left: 45%;
+	tr:nth-child(even) {background-color: #f2f2f2;}
+
+	@media (max-width: 500px) {
+		width: 70%;
+		height: 200px;
+    	margin-left: 31%;
+	}
+		
+		td {
+			.tdStylz()
+
+		}
+	
+}
+
+	th {
+		font-size: @header-font;		
+	}	
+
+}
+
+// Styling for workHistory table
+
+.workHistory {
+	padding-top: 50px;
+	width: 700px;
```

Comment:
Overall the styling is great!  Mostly responsive units,  I am impressed.  I think there were only 2 fixed pixel widths.  For the .workHistory and for the table.